### PR TITLE
Basic Client

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
Basic barchart.com API Client with support for prices (getQuotes)
and some other options methods.

See CP-3

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+AllCops:
+  TargetRubyVersion: 2.5
```
Comment (josephbridgwaterrowe) [inline .rubocop.yml]: Change this to 2.6?

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.6.3
```
Comment (josephbridgwaterrowe) [inline .ruby-version]: 2.6

File: barchart.gemspec [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  spec.files = `git ls-files -z`.split("\x0").reject do |f|
+    f.match(%r{^(test|spec|features)/})
+  end
+  spec.bindir = 'bin'
+  spec.executables << 'agent'
+  spec.require_paths = ['lib']
+
+  spec.add_runtime_dependency 'activesupport', '~> 6'
+  spec.add_runtime_dependency 'dotenv', '2.7.5'
```
Comment (josephbridgwaterrowe) [inline barchart.gemspec]: Do we need this?

File: barchart.gemspec [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  spec.add_runtime_dependency 'gli', '2.19.0'
```
Comment (josephbridgwaterrowe) [inline barchart.gemspec]: Or this?

**Review by josephbridgwaterrowe — COMMENTED** [review]
Cleanup.

**Review by cracell — APPROVED** [review]

**Pull request merged**